components/service/glean/README.md

```diff
@@ -101,6 +103,22 @@ Every application must provide a way to disable and re-enable data collection an
 controlled with the `glean.setUploadEnabled()` method. The application should
 provide some form of user interface to call this method.
 
+## Debugging products using glean
+Glean exports the [`GleanDebugActivity`](src/main/java/mozilla/components/service/glean/debug/GleanDebugActivity.kt)
+that can be used to toggle debugging features on or off. Users can invoke this special activity, at
+run-time, using the following [`adb`](https://developer.android.com/studio/command-line/adb) command:
+
+`adb shell am start -n [package.name]/mozilla.components.service.glean.debug.GleanDebugActivity [extra keys]`
+
+Where the `[package.name]` is the product's package name as exported in the manifest file (e.g.
+`org.mozilla.samples.glean` for the glean sample application) and `[extra keys]` is a list of
+extra keys to be passed to the debug activity. See the [documentation](https://developer.android.com/studio/command-line/adb#IntentSpec)
+for the command line switches used to pass the extra keys. These are the currently supported keys:
```

Comment:
Can you give an example of a full working command line (that i could just copy & paste)?